### PR TITLE
Update torch in eager mode CI pipeline

### PR DESCRIPTION
**Pull request by baijumeswani**
Addressing CG issue.

co-contributers: @souptc, @baijumeswani

### CODE REVIEW

File: orttraining/orttraining/eager/test/ort_tensor.py [hunk 1]
```diff
@@ -21,25 +21,29 @@ def test_is_ort_via_to(self):
...
         assert y.size()[0] == 100
 
+    @unittest.skip("Test fails with newest pytorch version.")
     def test_view(self):
         cpu_ones = torch.ones(2048)
         ort_ones = cpu_ones.to("ort")
         y = ort_ones.view(4, 512)
         assert y.size() == (4, 512)
 
+    @unittest.skip("Test fails with newest pytorch version.")
```
Comment (justinchuby) [inline orttraining/orttraining/eager/test/ort_tensor.py:39]: I would specify the version number for clarity. Would using `@unittest.expectedFailure` make sense, so when tests are fixed we know to resume them?

**Review by adrianlizarraga — COMMENTED** [review]
LGTM. Do we need to merge latest main commits to make sure CI still passes?

**Comment by baijumeswani** [comment]
I think it is OK to merge considering there has not been any active development on eager mode front.

File: orttraining/orttraining/eager/test/ort_tensor.py (hunk 1 above)
Comment (baijumeswani) [reply orttraining/orttraining/eager/test/ort_tensor.py:39]: Thanks Justin. These unit tests and the entire eager mode code is planned to be soon deprecated. We do not intend to re-enable them in the future.
I think this comment is fine for now, until @souptc and his team deprecated eager mode. Let me know if you strongly feel about updating this comment, will update it in that case.

**Review by adrianlizarraga — APPROVED** [review]

**Pull request merged**